docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -293,6 +298,79 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+## Sending and Receiving Custom Events
+
+Custom commands are a feature that enables the client application to talk to the server application.
```

Comment:
~ command**s are a** feature that enable**s** the ~

I'm not sure with the usage of plural  and singular in this sentence. Would you please check once again?